Services/UserService/Controllers/UserServiceController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using IDeleteUserUserService.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Razor.Language.Intermediate;
+using UserService.Commands;
+
+namespace UserService.Controllers
+{
+    [ApiController]
+
+    public class UserServiceController : ControllerBase
+    {
+        [Route("[controller]/deleteUser")]
+        [HttpGet]
+        public HttpStatusCode DeleteUserFromDataBaseService([FromServices] IDeleteUser command,[FromQuery] int userId)
```

Comment:
What does deleting user from service mean?

Reply:
> What does deleting user from service mean?

it's the Route

Reply:
agreed with Nikita it is a bad name for controller endpoint, **DeleteUser** will be enough
Check userId type, i'm not sure about **int**